documentation/getting-started-cli/source/debugging-with-gdb-lldb.rst

```diff
@@ -176,21 +184,77 @@ first, otherwise the message 'No generic function XXX was found.' will
 be shown.
 
 The second purpose of the helper script is to show Dylan objects in a
-more intuitive fashion. For example, lldb on its own will show most
-Dylan objects as plain hex values, for example::
+more intuitive fashion. LLDB on its own will show most Dylan objects
+as plain hex values, for example:
+
+.. code-block:: none
 
   (dylan_value) T33 = 0x0000000100d38060
   (dylan_value) T35_0 = 0x00007ffeefbfe360
   (dylan_value) Ustream_ = 0x0000000000000001
 
-With the helper, extra information is added to the right::
+With the helper, extra information is added to the right:
+
+.. code-block:: none
+
 
   (dylan_value) T33 = 0x0000000100c38060 {<symbol>: #"libraries-test-suite-app"}
   (dylan_value) T35_0 = 0x00007ffeefbfe370 {<simple-object-vector>: size: 1}
   (dylan_value) Ustream_ = 0x0000000000000001 {<integer>: 0}
 
 The summarizer support has to be added on a class-by-class basis, so
 some objects will show only the class name without further detail.
+Because LLDB was principally a C debugger, some concepts such as
+pointers need to be considered. Given a Dylan class like:
+
+.. code-block:: none
+
+  define class <element>(<object>)
```

Comment:
Nit: there is normally a space between the definition name and the superclass list (or argrlist).

Reply:
done